### DISCUSSION TITLE
Is there a REST API to create a new custom workload item?

### DISCUSSION THREAD

**Discussion by q-benwillis** [microsoft/Microsoft-Fabric-workload-development-sample#160] [Help] [answered]
In our Fabric Workload front end we can create a new workload item using the workload-client `ItemCrudAPI` `createItem` method in the front end.

Is there a way to create a new workload item using a REST API from outside our workload?

We'd like to automate some of testing of our public third party workload and part of this is testing the backend of our workload in isolation without using the front end. To do this we would need the ability to programmatically create a new workload item of our custom type using a REST API and not using the workload client.

**Comment by vadimazarov** [comment, marked as answer]
(unlike my previous answer :) )
 - Create (of custom workload items) via REST API is not yet supported
